api/v1/models/endpoint_policy_enabled.go

```diff
@@ -31,14 +31,23 @@ const (
 
 	// EndpointPolicyEnabledBoth captures enum value "both"
 	EndpointPolicyEnabledBoth EndpointPolicyEnabled = "both"
+
+	// EndpointPolicyEnabledAuditIngress captures enum value "audit-ingress"
+	EndpointPolicyEnabledAuditIngress EndpointPolicyEnabled = "audit-ingress"
+
+	// EndpointPolicyEnabledAuditEgress captures enum value "audit-egress"
+	EndpointPolicyEnabledAuditEgress EndpointPolicyEnabled = "audit-egress"
+
+	// EndpointPolicyEnabledAuditBoth captures enum value "audit-both"
+	EndpointPolicyEnabledAuditBoth EndpointPolicyEnabled = "audit-both"
```

Comment:
It looks like this PR didn't include the corresponding changes made to `api/v1/openapi.yaml` from which these consts were generated using `make generate-api`. When I rerun `make generate-api`, these consts are gone :( Same for the hunk below.

Reply:
Adressed in #12652. I'll also make a PR adding a GitHub action to check the generated API (similar to what we already do for the k8s APIs).